### PR TITLE
add 2 new error type for UDP connectivity

### PR DESCRIPTION
**Pull request by xiaoxiye2022**
add two error code for UDP connectivity

### CODE REVIEW

**Comment by xiaoxiye2022** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 1]
```diff
@@ -3098,6 +3098,32 @@
...
+            {
+                "Id": "59abcc02-0b31-48a0-a44b-052a3d123267",
+                "ApiChange": "Property",
+                "ChangedApiName": "udpConnectivityCheckTurnUrlNotAllowListed",
+                "ChangeType": "Addition",
+                "Description": "Added the **udpConnectivityCheckTurnUrlNotAllowListed** member to the [cloudPcOnPremisesConnectionHealthCheckErrorType](https://learn.microsoft.com/en-us/graph/api/resources/cloudpconpremisesconnectionhealthcheck?view=graph-rest-beta#cloudpconpremisesconnectionhealthcheckerrortype-values) resource.",
+                "Target": "cloudPcOnPremisesConnection"
+            },
+            {
+                "Id": "1625de52-ebc2-496e-a15c-87474fcc0e0e",
```
Comment (RuiHou105) [inline changelog/Microsoft.CloudManagedDesktop.json:3113]: The three ids should be the same. Please fix it.

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 2]
```diff
@@ -3098,6 +3098,32 @@
...
+      },
+      {
+        "ChangeList": [
+            {
+                "Id": "59abcc02-0b31-48a0-a44b-052a3d123267",
+                "ApiChange": "Property",
+                "ChangedApiName": "udpConnectivityCheckTurnUrlNotAllowListed",
+                "ChangeType": "Addition",
+                "Description": "Added the **udpConnectivityCheckTurnUrlNotAllowListed** member to the [cloudPcOnPremisesConnectionHealthCheckErrorType](https://learn.microsoft.com/en-us/graph/api/resources/cloudpconpremisesconnectionhealthcheck?view=graph-rest-beta#cloudpconpremisesconnectionhealthcheckerrortype-values) resource.",
+                "Target": "cloudPcOnPremisesConnection"
```
Comment (RuiHou105) [inline changelog/Microsoft.CloudManagedDesktop.json:3110]: I believe the target should be `cloudPcOnPremisesConnectionHealthCheckErrorType`. Please help fix.

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
@xiaoxiye2022 
1.Please resolve comments in the PR.
2.Please create a roadmap workitem and attach the workitem **number** in comment. Here is an [example ](https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/14460/)for you. If you don't have access, please do these firstly
- Request access via [Improved ADO Access - 1ES Wikirequest to join](https://www.1eswiki.com/wiki/Improved_ADO_Access) .
- Join the Microsoft Graph Roadmap Users ([graphroadmap@microsoft.com](mailto:graphroadmap@microsoft.com)) security group on [idweb](https://idwebelements/GroupManagement.aspx?Group=graphroadmap&Operation=join) .

File: changelog/Microsoft.CloudManagedDesktop.json (hunk 1 above)
Comment (xiaoxiye2022) [reply changelog/Microsoft.CloudManagedDesktop.json]: fixed

File: changelog/Microsoft.CloudManagedDesktop.json (hunk 2 above)
Comment (xiaoxiye2022) [reply changelog/Microsoft.CloudManagedDesktop.json]: fixed

**Comment by Jarbas-MSFT** [comment]
Hi @Lauragra, this PR is ready for final review.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**